docs/csharp/create-a-nested-group.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Create a nested group"
```

Comment:
We seem to be inconsistent here. Some of the topics under C# Concepts (where these topics are) have "| C# Guide", but many just have the title without the pipe.